RandomCats/RandomCatAPI.swift

```diff
@@ -1,13 +1,14 @@
 //
-//  RandomDogAPI.swift
-//  RandomDogs
+//  RandomCatAPI.swift
+//  RandomCats
 //
-//  Created by Spud on 4/23/22.
+//  Created by Quang on 6/4/22.
```

Comment:
I think it's better to leave the "Created By" but add a "Edited By" :D you don't have to for this project, but in the future some developers might not take kindly to having their name erased haha no problemo for this project tho :]

Reply:
Haha thank you so much! Fully noted!